deploy/keycloak-install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+ssotemplate=$(oc get templates -n openshift -o=custom-columns=NAME:.metadata.name --ignore-not-found | grep sso73-x509-postgresql-persistent)
+
+if [[ ${ssotemplate} ]]; then
+   echo "Installing Keycloak ..."
+   echo "Copying the template ${ssotemplate} to json file ..."
+   oc get template ${ssotemplate} -n openshift -o json > ${ssotemplate}.json
+   echo "The template json file is: " ${ssotemplate}.json
+   echo "Modifying the josn file with postgresql version 9.6 and mountpath to resolve known issues on postgresql ..."
+   sed -i 's/"9.5"/"9.6"/g' ${ssotemplate}.json
```

Comment:
How confident are you that this template will always be present, and that it won't change in such a way that will make these `sed` commands break?  I will be honest and say that I am not very familiar with OpenShift templates and what their lifecycle is.  Is this likely to work in OCP 4.3?  

I am wondering if a better approach would be to retrieve the JSON once, modify it as needed, and then check in those modifications into our `optional.yaml` file.

Reply:
fixed